foundation-extensions/experience-platform-extension/release-notes.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## Sept 3, 2021
+
+### Adobe Experience Platform Edge Network Launch extension v1.0.12
+
+- New `Forward to Edge Network` action to be used with Mobile Core Application Lifecycle events.
```

Comment:
Should we just say what this action does instead of `used with Mobile Core Application Lifecycle events.`

Reply:
I would say we need more investigation for other events, so I prefer keeping it related to Lifecycle events for now.